Aplib.Core/Belief/BeliefSets/BeliefSet.cs

```diff
@@ -1,14 +1,18 @@
-﻿using Aplib.Core.Belief.Beliefs;
+// This program has been developed by students from the bachelor Computer Science at Utrecht
+// University within the Software Project course.
+// Copyright Utrecht University (Department of Information and Computing Sciences)
+
+using Aplib.Core.Belief.Beliefs;
 using System.Linq;
 
 namespace Aplib.Core.Belief.BeliefSets
 {
     /// <summary>
     /// This class can be inherited to define a set of beliefs for an agent.
-    /// All <i>public fields</i> defined in the inheriting class that implement <see cref="IBelief" /> 
+    /// All <i>public fields</i> defined in the inheriting class that implement <see cref="IBelief" />
     /// are automatically updated when calling <see cref="UpdateBeliefs" />.
     /// </summary>
```

Comment:
Interesting, what were your thoughts behind this?

Reply:
I have removed some invisible characters that were present in the code for some reason. No clue why they were there, this probably must be such a thing.

Reply:
Lol I responded to the wrong line. I wanted to react to the abstract thing haha, damn I was tired

Reply:
Probably was whitespace at the end